dapp/src/components/AccountListener.js

```diff
@@ -53,6 +54,102 @@ const AccountListener = (props) => {
     }
   }, [active, prevActive, account, prevAccount])
 
+  const pollOnce = async (contracts) => {
```

Comment:
should the name of this function be `pullOnce` instead of `pollOnce`?

Reply:
yeah good point. polling is a more recurrent process.. i'll update it. 

Right now we have `loadData`, `pollOnce`, `subscribeToEvents`. Maybe we can do this?
- `pollData`, `loadDataOnStart`,  `subscribeToEventData`
- 
In run time they are all treated differently. pollData is for recurrent polling. loadDataOnStart is a one time pull. subscribeToEventData is for subscribing to events.

Reply:
Ok for now I renamed pollOnce to pullOnce. It looks fine to me

Reply:
hey yeah I like the renames you suggest. Go for it 👍